dix/cursor.c

```diff
@@ -504,7 +503,7 @@ CreateRootCursor(void)
                                   serverClient, DixReadAccess);
     if (err != Success)
         return NullCursor;
-    if (AllocGlyphCursor(fontID, 0, fontID, 1, 0, 0, 0, ~0, ~0, ~0,
+    if (AllocGlyphCursor(fontID, 0, fontID, 1, 0, 0, 0, 0xFFFF, 0xFFFF, 0xFFFF,
```

Comment:
Hmm, if in theory on some system short is more than 2 bytes, then this can be a problem, the old version is safer. Not likely to happen in practice though..

Reply:
changed it to using that - but it needs extra typecast.